src/app/(search)/page.tsx

```diff
@@ -1,3 +1,58 @@
-export default function Page() {
-  return <p>home</p>;
+import Link from "next/link";
+
+import { HorizontalChefList } from "@/components/horizontal-chef-list/horizontal-chef-list";
+import { ChefListItemWithRecipeCountType, VerticalChefList } from "@/features/chefs";
+import { getRecipesFavoritedRecently, HorizontalRecipeList } from "@/features/recipes";
+
+export default async function Page() {
+  // TODO: DBから取得する
+  const popularChefs: {
+    id: string;
+    name: string;
+    imageSrc: string;
+  }[] = [...Array(10)].map((_, i) => ({
+    id: `${i}`,
+    name: "シェフ名",
+    imageSrc: "/images/chefs/4j4vh2DXrNxPBgSC.jpeg",
+  }));
+
+  const popularRecipeList = await getRecipesFavoritedRecently({ query: "" });
+
+  // TODO: DBから取得する
+  const chefs: ChefListItemWithRecipeCountType[] = [...Array(10)].map((_, i) => ({
+    id: `${i}`,
+    name: "シェフ名",
+    image: "/chefs/4j4vh2DXrNxPBgSC.jpeg",
+    createdAt: new Date("2021-09-01T00:00:00.000Z"),
+    updatedAt: new Date("2021-09-01T00:00:00.000Z"),
+    recipeCount: i,
+    profileText: "シェフのプロフィールです。シェフのプロフィールです。シェフのプロフィールです。",
+  }));
+
+  return (
+    <>
+      <div className="flex flex-col gap-y-4">
+        <p className="text-xl text-mauve-12">注目のシェフ</p>
+        <HorizontalChefList chefs={popularChefs} />
+      </div>
+      <div className="mt-12 flex flex-col gap-y-[10px]">
+        <div className="flex items-center justify-between">
+          <p className="text-xl text-mauve-12">話題のレシピ</p>
+          <Link href="/search/recipe" className="text-mauve-9">
+            もっと見る
+          </Link>
+        </div>
+        <HorizontalRecipeList recipeList={popularRecipeList} />
+      </div>
+      <div className="mt-12 flex flex-col gap-y-4">
+        <div className="flex items-center justify-between">
+          <p className="text-xl text-mauve-12">シェフ</p>
+          <Link href="/search/chef" className="text-mauve-9">
+            もっと見る
+          </Link>
+        </div>
+        <VerticalChefList chefList={chefs} />
+      </div>
+    </>
```

Comment:
見た目は変わらないのですが、以下のようにするのはどうでしょうか？
なんとなくUIの構造がわかりやすくなる気がしたので、提案させていただきました。
```suggestion
    <section className="flex flex-col gap-y-12">
      <section className="flex flex-col gap-y-4">
        <h2 className="text-xl text-mauve-12">注目のシェフ</h2>
        <HorizontalChefList chefs={popularChefs} />
      </section>
      <section className="flex flex-col gap-y-2.5">
        <div className="flex items-center justify-between">
          <h2 className="text-xl text-mauve-12">話題のレシピ</h2>
          <Link href="/search/recipe" className="text-mauve-9">
            もっと見る
          </Link>
        </div>
        <HorizontalRecipeList recipeList={popularRecipeList} />
      </section>
      <section className="flex flex-col gap-y-4">
        <div className="flex items-center justify-between">
          <h2 className="text-xl text-mauve-12">シェフ</h2>
          <Link href="/search/chef" className="text-mauve-9">
            もっと見る
          </Link>
        </div>
        <VerticalChefList chefList={chefs} />
      </section>
    </section>
```

Reply:
こちらのが良さそうので修正します！